Source/loadsave.cpp

```diff
@@ -1746,6 +1746,10 @@ void SaveHeroItems(PlayerStruct *pPlayer)
 	SaveItems(&file, pPlayer->SpdList, MAXBELTITEMS);
 }
 
+// 256 kilobytes + 3 bytes (demo leftover) for file magic (262147)
+// final game uses 4-byte magic instead of 3
+#define FILEBUFF ((256 * 1024) + 3)

```

Comment:
Isn't it common to have the defines closer to the top of the file?

Reply:
if its only used in a specific function i think it makes more sens to have it next to it